cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs

```diff
@@ -375,16 +389,84 @@ instance (ShelleyBasedEra era, Core.Value era ~ Coin)
              , "addrs"   .= addrs
              ]
 
+instance ToJSON (MA.Value era) where
+  toJSON = panic "TODO @node team: implement it. Should this be defined in cardano-ledger-specs?"
+
+-- TODO @node team: does this make sense? Note that the object can be empty.
+instance ToJSON MA.ValidityInterval where
```

Comment:
We're currently using an intermediate type for this: https://github.com/input-output-hk/cardano-node/pull/2010/commits/f49a229a286baced9397d8a8cadf890a4c310a6d#diff-8595c8e945f3567404d3c90c4a43eaaf9dfb94aebecfac4810d39a20fc04b0ecR315